app/models/admin.rb

```diff
@@ -0,0 +1,2 @@
+class Admin < ApplicationRecord
```

Comment:
Is this required or was this created as part of bug fix exploration? Would love to chat through how you solved this.